test/functions.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { FilterFunctionTestData, testFilterFunction } from './helper/test-function';
+import { loadFixtureFile } from './helper/util';
+
+import {
+	albumArtistFromArtist,
+	decodeHtmlEntities,
+	fixTrackSuffix,
+	normalizeFeature,
+	removeZeroWidth,
+	replaceNbsp,
+	removeCleanExplicit,
+	removeLive,
+	removeRemastered,
+	removeVersion,
+	removeParody,
+	removeFeature,
+	youtube,
+} from '../src/functions';
+
+function loadFunctionFixtureFile(functionId: string): FilterFunctionTestData[] {
+	const fixtures = loadFixtureFile<FilterFunctionTestData>(`functions/${functionId}`);
+
+	for (const fixture of fixtures) {
+		if (typeof fixture.description !== 'string') {
+			throw new Error(`Missing description in test case for '${functionId}'`);
+		}
+		if (typeof fixture.funcParameter !== 'string') {
+			throw new Error(`Missing funcParameter in test case for '${functionId}'`);
+		}
+		if (typeof fixture.expectedValue !== 'string') {
+			throw new Error(`Missing expectedValue in test case for '${functionId}'`);
+		}
+	}
+
+	return fixtures;
+}
+
+const functionsToTest = {
+	'album-artist-from-artist': albumArtistFromArtist,
+	'decode-html-entities': decodeHtmlEntities,
+	'fix-track-suffix': fixTrackSuffix,
+	'normalize-feature': normalizeFeature,
+	'remove-zero-width': removeZeroWidth,
+	'replace-nbsp': replaceNbsp,
+	'remove-clean-explicit': removeCleanExplicit,
+	'remove-live': removeLive,
+	'remove-remastered': removeRemastered,
+	'remove-version': removeVersion,
+	'remove-parody': removeParody,
+	'remove-feature': removeFeature,
+	'youtube': youtube,
```

Comment:
I noticed ESLint shows an error for this line when I run `npm run lint`:
```
51:2  error  Expected property shorthand  object-shorthand
```

 Could you suppress it (for this line or for the entire file - both ways are fine)?

I'm not sure why GA runner does not produce an error. Maybe it's related to https://github.com/web-scrobbler/metadata-filter/commit/e1fa65d53683bd42e78483a67f122d24185a2ad0.

Reply:
I think a better approach might be to change how the rule is configured everywhere. ESLint's `object-shorthand` rule supports a mode named `consistent-as-needed`, which I think would be most appropriate to cover this situation. Thoughts?

Reply:
Yes, that's better.